### ISSUE TITLE
GPS continous values

### ISSUE THREAD

**Issue by Lallering** [malcommac/SwiftLocation#307]
I try to subscribe to GPS updates as it comes in the demo and after initializing it gets canceled

![image](https://user-images.githubusercontent.com/33471019/99718345-570b6380-2a70-11eb-9c68-9213b2017f0f.png)

i get this on console:

![image](https://user-images.githubusercontent.com/33471019/99718424-730f0500-2a70-11eb-8c55-097e17bd6a25.png)

I don't know what eviction policy is

**Comment by hamzaozturk** [comment]
Currently, configuring service options using block is buggy. You can use the code below.

First, define your service options in a separate variable.
```
let serviceOptions = GPSLocationOptions()
serviceOptions.subscription = .continous
serviceOptions.accuracy = .block
serviceOptions.minDistance = 7
serviceOptions.activityType = .otherNavigation
```

Then, initialize your request like below:

```
let gpsRequest = SwiftLocation.gpsLocationWith(serviceOptions).then {
...
}
```

**Comment by malcommac** [comment]
Fixed in 5.0.1 THanks.
